### DISCUSSION TITLE
Problem in running the example

### DISCUSSION THREAD

**Discussion by YuliangS** [MarsalekGroup/aml#27] [Q&A]
Hi
I meet a problem when I run "run-QbC.py" 

It said: "ValueError: cannot reshape array of size 7 into shape (2,3)"

**Comment by OndrejMarsalek** [comment]
That simply means that an array of size 7 can't be reshaped into a 2x3 array, which is, indeed, true. Without more information, it is impossible to tell why this happens. Is this the `run-QbC.py` from the `from-trajectroy-to-model` example directory? Did you modify it at all? Can you provide a full traceback with source files and line numbers?

**Reply by YuliangS** [reply]
Thank you so much for your reply.

Yes, after AIMD calculations, I ran ”run-QbC.py“ directly without any modifications.

Here it the full traceback:

![WeChat3af57b745a9686393508d69c1570bc86](https://github.com/MarsalekGroup/aml/assets/105383439/9a8f4e0c-6746-4fdd-87cf-fe554f1c5091)

**Reply by OndrejMarsalek** [reply]
Unfortunately, at the moment, we do not really have any real error handling (although it is coming up). You can see that the commands that were launched by AML (n2p2 training in this case) finished with a non-zero return code, indicating an issue. The `ValueError` then comes from an attempt to read the output of those failed calculations. It is not possible to tell why they failed, you will have to investigate - the directories are left there, so you can just have a look. Try running the same n2p2 calculation in the same environment to examine why it fails.

**Comment by YuliangS** [comment]
Thank you so much for the suggestion.
